newscorp-ghfb/newskit
# fix(PPDSR-2081): fix broken link in site

PPDSC-2081

**What**

1. Background - why this is needed - Link on page was returning a 404 page when you click on it.
2. What did you do - changes from relative to absolute path
3. What does the reviewers should expect - link to work.

**I have done:**

**I have tested manually:**
- [ ] The feature's functionality is working as expected on Chrome, Firefox, Safari and Edge
- [ ] The screen reader reads and flows through the elements as expected.




**Before:**

<img width="1418" alt="Screenshot 2022-04-12 at 11 04 16 am" src="https://user-images.githubusercontent.com/11298045/162939977-b5cba3e1-0c31-4e04-b0e1-b0a56033f8a1.png">

**After:**

<img width="1422" alt="Screenshot 2022-04-12 at 11 04 34 am" src="https://user-images.githubusercontent.com/11298045/162939994-4368bd19-0f13-4513-93d5-ef4f53d238c2.png">

**Who should review this PR:**


**How to test:**

Go to [http://ncu-newskit-docs.s3-website-eu-west-1.amazonaws.com/ppdsc-2081-fix-link/theme/presets/style-presets/#using](http://ncu-newskit-docs.s3-website-eu-west-1.amazonaws.com/ppdsc-2081-fix-link/theme/presets/style-presets/#using) and click on [learn more about using a theme in code](http://ncu-newskit-docs.s3-website-eu-west-1.amazonaws.com/ppdsc-2081-fix-link/theme/theming/using-a-theme/) link. You should go to (http://ncu-newskit-docs.s3-website-eu-west-1.amazonaws.com/ppdsc-2081-fix-link/theme/theming/using-a-theme/)[http://ncu-newskit-docs.s3-website-eu-west-1.amazonaws.com/ppdsc-2081-fix-link/theme/theming/using-a-theme/]
